### PR TITLE
Adding DANM

### PR DESCRIPTION
**Pull request by CsatariGergely**
Adding a new item to the list of cluster networking solutions.

Signed-off-by: Gergely Csatari <gergely.csatari@nokia.com>

### CODE REVIEW

File: content/en/docs/concepts/cluster-administration/networking.md [hunk 1]
```diff
@@ -152,6 +152,18 @@ CNI-Genie also supports [assigning multiple IP addresses to a pod](https://githu
 
 [Contrail](http://www.juniper.net/us/en/products-services/sdn/contrail/contrail-networking/), based on [OpenContrail](http://www.opencontrail.org), is a truly open, multi-cloud network virtualization and policy management platform. Contrail / OpenContrail is integrated with various orchestration systems such as Kubernetes, OpenShift, OpenStack and Mesos, and provides different isolation modes for virtual machines, containers/pods and bare metal workloads.
 
+### DANM
+
+[DANM](https://github.com/nokia/danm) is a networking solution for telco workloads running in a Kubernetes cluster. It's built up from the following components:
+
+   * a CNI plugin capable of provisioning IPVLAN interfaces with advanced features
```
Comment (steveperry-53) [inline content/en/docs/concepts/cluster-administration/networking.md]: Suggestion: Start each bullet with an upper-case letter. That will be consistent with most of the other lists in the doc set.

* A CNI plugin capable of ...

File: content/en/docs/concepts/cluster-administration/networking.md [hunk 2]
```diff
@@ -152,6 +152,18 @@ CNI-Genie also supports [assigning multiple IP addresses to a pod](https://githu
...
+   * an in-built IPAM module with the capability of managing multiple, cluster-wide, discontinuous L3 networks and provide dynamic, static, or no IP allocation scheme on-demand
+   * a CNI metaplugin capable of attaching multiple network interfaces to a container, either through its own CNI, or through delegating the job to any of the popular CNI solution like SRI-OV, or Flannel in parallel
```
Comment (steveperry-53) [inline content/en/docs/concepts/cluster-administration/networking.md]: ... discontinuous L3 networks and providing a dynamic ...

**Comment by steveperry-53** [comment]
@CsatariGergely Looks good. Just a couple of comments in the text.

**Comment by CsatariGergely** [comment]
Thanks @steveperry-53 for the comments. I've applied the needed changes.

**Pull request merged**